.github/workflows/release.yml

```diff
@@ -34,23 +44,6 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Check existing tags
-        run: |
-          set +e
-          RECREATE_TAGS=${{ github.event.inputs.forceRecreateTags }}
```

Comment:
so... we don't want to prevent accidental re-releases anymore? Why disable this check?